RosettaCore/src/main/java/com/hubspot/rosetta/RosettaBinder.java

```diff
@@ -15,6 +16,24 @@ public interface Callback {
     void bind(String key, Object value);
   }
 
+  public void bindList(ArrayNode node, String field, Consumer<Object> valueConsumer) {
+    for (JsonNode value : node) {
+      if (!field.isEmpty()) {
+        if (!value.has(field)) {
+          throw new IllegalArgumentException(String.format("Field %s does not exist in elements of input array", field));
```

Comment:
I might be wrong, but I don't think we flex the case where we have a field that's missing from the pojo in a test.

Reply:
Ah yep, added a test for that 👍